source/content/guides/domains/05-custom-domains.md

```diff
@@ -100,9 +100,7 @@ For any domain that has been added that is pending verification, clicking on the
 You can get a high-level status view for all custom domains connected to a given environment via Terminus using the [`https:info` command](/terminus/commands/https-info). Domains that are pending verification will have the "Verification Pending" status returned as part of the Terminus `https:info` command.
 
 ### Can I opt-out of Domain Verification?
```

Comment:
Todo: Alll other account tiers for professional workspaces can request exemption by reaching out to support. still only the workspace level, no exemptions for a user or single site